csds-client/client/v3/client.go

```diff
@@ -0,0 +1,325 @@
+// Package client/v3 implements the client interface for v3 transport api version
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"envoy-tools/csds-client/client"
+	clientUtil "envoy-tools/csds-client/client/util"
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	csdspb_v3 "github.com/envoyproxy/go-control-plane/envoy/service/status/v3"
+	envoy_type_matcher_v3 "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+)
+
+// ClientV3 implements the Client interface
+type ClientV3 struct {
+	clientConn *grpc.ClientConn
+	csdsClient csdspb_v3.ClientStatusDiscoveryServiceClient
+
+	nodeMatcher []*envoy_type_matcher_v3.NodeMatcher
+	metadata    metadata.MD
+	opts        client.ClientOptions
+}
+
+// field keys in NodeMatcher
+const gcpProjectNumberKey string = "TRAFFICDIRECTOR_GCP_PROJECT_NUMBER"
```

Comment:
You can define them in one const block

const (
 firstConst = "ggg"
 secondConst = "zzz"
)